.github/workflows/python-check.yaml

```diff
@@ -74,16 +73,18 @@ jobs:
         uses: astral-sh/setup-uv@v6
         with:
           enable-cache: true
-          working-directory: icechunk-python
+          python-version: ${{ env.PYTHON_VERSION }}
 
       - name: Install dependencies
         shell: bash
         working-directory: icechunk-python
         run: |
           set -e
-          # Create virtual environment and install the built wheel
           uv venv
-          uv pip install icechunk['test'] --find-links dist
+          source .venv/bin/activate
+          python --version
+          WHEEL=$(ls dist/*-cp311-*.whl)
+          uv pip install "${WHEEL}[test]"
```

Comment:
Somewhat horrifyingly this was previously not installing the source icechunk. I'm a little alarmed I didn't catch that earlier.

I have confirmed that it does not. This was not the cause of the failures, but a separate also important thing I discovered in the process.